ietf/group/models.py

```diff
@@ -391,6 +393,32 @@ def formatted_ascii_email(self):
     def formatted_email(self):
         return formataddr((self.person.plain_name(), self.email.address))
 
+    def started_in_role(self):
+        EPOCH = datetime.datetime(2011, 12, 9, 20, 0, tzinfo=pytz.utc)
+        history = (
+            Person.objects.filter(
+                pk=self.person.pk,
+                rolehistory__name=self.name,
+                rolehistory__group__group=self.group,
+            )
+            .values_list("rolehistory__group__time")
+            .order_by("rolehistory__group__time")
+            .first()
+        )
+        print("history", self.person.plain_name(), history)
+        current = (
+            Person.objects.filter(
+                pk=self.person.pk, role__name=self.name, role__group=self.group
+            )
+            .values_list("role__group__time")
+            .first()
+        )
```

Comment:
What is this trying to accomplish that's different than just taking self.group.time?